api-tests/features/repeat-fraud-check.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Repeat fraud check journeys
+
+  @Build
+  Scenario: Fraud 6 Months Expiry + Name Update
```

Comment:
You should be able to use a [Background](https://cucumber.io/docs/gherkin/reference/#background)  to reduce duplication :)

Reply:
oh nice thanks didn't know about this :)